doc/user-guide/user-guide.rst

```diff
@@ -125,26 +192,31 @@ After that, proceed to updating the required versions for each package.
 
 For the :ref:`core packages <Core-packages>`, the version will be set in agreement with all the other subsystems.
 Furthermore, one may be interested in updating the DM stack version (``lsstsqre``) as well.
-All the other packages can either be inspected in github for the latest version or confirmed with the main developer.
+All the other packages can either be inspected in github for the latest version or confirmed with their main developer.
 In some cases, this process can be a bit time consuming as one needs to verify which version should be deployed with several different sources.
 
 Once ``cycle/cycle.env`` is updated, you should be ready to start a build.
-For that, commit and push your changes to GitHub on a ticket branch.
+For that, commit and push your changes.
 Make sure to describe the changes you have made on the commit message and also update the version history.
 
 Once the changes are pushed to GitHub the branch will appear in the `cycle build jenkins job`_.
-Select the branch you are working on in the Branch tab and select Build with Parameters on the left-hand side, which will take you to the :ref:`fig-jenkins-build-with-parameters`.
+Select the branch you are working on in the Branch tab and select ``Build with Parameters`` on the left-hand side, which will take you to the :ref:`fig-jenkins-build-with-parameters`.
 
-.. _cycle build jenkins job: https://tssw-ci.lsst.org/view/CSC_Docker/job/cycleBuild/
+.. _cycle build jenkins job: https://tssw-ci.lsst.org/view/CycleBuild/job/cycleBuild/
 
 .. figure:: /_static/JenkinsBuildWithParameters.png
    :name: fig-jenkins-build-with-parameters
    :target: ../_images/SingleProcessFig.png
-   :alt: Jenkins build with parameters
+   :alt: Jenkins build with parameters page
 
    Jenkins build with parameters page.
 
-The build is divided into different steps.
+.. _Building-a-new-Cycle-or-Revision-Build-Steps:
+
+Build Steps
+^^^^^^^^^^^
+
+The build is sub-divided into different steps made to build the system from the ground up.
```

Comment:
This sentence reads a bit oddly. Maybe something like: The build is divided into steps to start from building base images up to deployable component images.